docs/installation.md

```diff
@@ -30,6 +30,7 @@ Works on macOS, Linux, and Windows, with support for both x86_64 and arm64 archi
     | [EasyOCR](https://github.com/JaidedAI/EasyOCR) | Default in Docling or via `pip install easyocr`. | `EasyOcrOptions` |
     | Tesseract | System dependency. See description for Tesseract and Tesserocr below.  | `TesseractOcrOptions` |
     | Tesseract CLI | System dependency. See description below. | `TesseractCliOcrOptions` |
+    | PaddleOCR | Extra feature not included in Default Docling installation can be installed via `pip install paddlepaddle paddleocr` | `PaddleOcrOptions` |
```

Comment:
@Swaymaw  Is paddleocr now supporting macos with M-chips? Last time we tried it out, it required some complicated installation with nightly builds.

Reply:
Yes I am currently working on a M3 Macbook Pro and it is working just fine, I used the same commands mentioned here to setup my environment and didn't face any issues.